runs/graphql/server.js

```diff
@@ -44,7 +49,8 @@ const resolvers = {
           markdown: parsed.body,
           content: marked(parsed.body),
           date,
-          type: parsed.attributes.type || ""
+          type: parsed.attributes.type || "",
+          presentations: parsed.attributes.speakers || []
```

Comment:
There is some inconsistentsy in that we call it speakers in the markdown, that is because later there will be a `type Speaker` that will be a node on `type Presentation` and ideally we would have a database of speakers.